components/Error.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: bold;
+  font-size: 1.5rem;
+`;
+
+type ErrorProps = {
+  message: string;
+};
+
+export const Error: React.FC<ErrorProps> = ({ message }) => (
```

Comment:
Add this beast to next's 404 and Error page. https://nextjs.org/docs/advanced-features/custom-error-page 

You will find it helpful in your future endeavors very soon